app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -581,7 +646,9 @@ class StudentForm extends React.Component {
                 <FormControl
                   componentClass="input"
                   placeholder="Address"
+                  id="address"
                   name="address"
+                  onFocus={(event) => this.handleAddressChange(event)}
                   onChange={(event) => this.handleChange(event)}/>
```

Comment:
actually you can take out the `onFocus` binding and just move `this.handleAddressChange` in the `onChange` instead